docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -0,0 +1,3023 @@
+7. Now select the following plugin from the ”KNIME Community Contributions - Cheminformatics” category
+   - RDKit KNIME integration
+8. Click on **Next**  and follow the instructions and after a restart of KNIME the dependencies will be installed.
+
+In addition, we need to install R for the statistical downstream analysis. Choose the directory that matches your
+operating system, double-click the R installer and follow the instructions. We recommend to use the default settings
+whenever possible. On macOS you also need to install XQuartz from the same directory.
+
+Afterwards open your R installation. If you use Windows, you will find an ”R x64 3.6.X” icon on your desktop. If you use
+macOS, you will find R in your Applications folder. In R type the following lines (you might also copy them from the file
+**R** > **install_R_packages.R** folder on the USB stick):
+
+```r
+install.packages('Rserve',,"http://rforge.net/",type="source")
+install.packages("Cairo")
+
+install.packages("devtools")
+install.packages("ggplot2")
+install.packages("ggfortify")
+
+if (!requireNamespace("BiocManager", quietly = TRUE))
+  install.packages("BiocManager")
+
+BiocManager::install()
+BiocManager::install(c("MSstats"))
+```
+In KNIME, click on **KNIME** > **Preferences**, select the category **KNIME** > **R** and set the ”Path to R Home” to
+your installation path. You can use the following settings, if you installed R as described above:
+
+- Windows: `C: \Program Files \R \R-3.6.X (where X is the version you used to install the above libraries)`
+- macOS: `/Library/Frameworks/R.framework/Versions/3.6/Resources`
+
+You are now ready to install the OpenMS nodes.
+- Open KNIME.
+- Click on **Help** > **Install New Software**
+
+We included a custom KNIME update site to install the OpenMS KNIME plugins from the USB stick. If you do not have a stick
+available, please see below.
+
+- In the now open dialog choose **Add** (in the upper right corner of the dialog) to define a new update site. In the
+  opening dialog enter the following details.
+
+  Name: OpenMS 2.7 UpdateSite
+  Location: `file:/KNIMEUpdateSite/2.7.0/`
+- After pressing **OK** KNIME will show you all the contents of the added Update Site.
+
+```{note}
+From now on, you can use this repository for plugins in the **Work with**: drop-down list.
+```
+
+- Select the OpenMS nodes in the ”Uncategorized” category and click **Next**.
+- Follow the instructions and after a restart of KNIME the OpenMS nodes will be available in the Node repository under
+  “Community Nodes”.
+
+Alternatively, you can try these steps that will install the OpenMS KNIME plugins from the internet. Note that download
+can be slow.
+
+- In the now open dialog, choose **Add** (in the upper right corner of the dialog) to define a new update site. In the
+  opening dialog enter the following details.
+
+  Name: OpenMS 2.5 UpdateSite
```

Comment:
how can we deal with these version numbers? just a few lines above 2.7.0 is mentioned, here it's 2.5.
Since this refers to nightly, 2.7 would be more accurate here as well